chainio/clients/avsregistry/reader.go

```diff
@@ -384,8 +385,9 @@ func (r *AvsRegistryChainReader) QueryExistingRegisteredOperatorPubKeys(
 	operatorPubkeys := make([]types.OperatorPubkeys, 0)
 
 	// eth_getLogs is limited to a 10,000 range, so we need to iterate over the range
```

Comment:
can you move this comment above to where you created the `QueryBlockRange = 10_000` const?

Reply:
Done